src/renderer/components/ft-list-lazy-wrapper/ft-list-lazy-wrapper.js

```diff
@@ -70,10 +64,9 @@ export default defineComponent({
           // hide livestreams
           return false
         }
-
         if (this.hideUpcomingPremieres &&
             // Observed for premieres in Local API Channels.
-            (data.durationText === 'PREMIERE' ||
+            (data.premiereDate != null ||
```

Comment:
Since Invidious API is mentioned might as well
https://docs.invidious.io/api/common_types/#videoobject
```suggestion
            (data.premiereDate != null ||
              // Invidious API
              // `premiereTimestamp` only available on premiered videos
              // https://docs.invidious.io/api/common_types/#videoobject
              data.premiereTimestamp != null ||
```